website/content/docs/configuration/server.mdx

```diff
@@ -118,38 +118,24 @@ server {
   example section](#configuring-scheduler-config) for more details
   `default_scheduler_config` was introduced in Nomad 0.10.4.
 
-- `heartbeat_grace` `(string: "10s")` - Specifies the additional time given as a
-  grace period beyond the heartbeat TTL of nodes to account for network and
-  processing delays as well as clock skew. This is specified using a label
-  suffix like "30s" or "1h".
-
-- `license_path` `(string: "")` - Specifies the path to load a Nomad Enterprise
-  license from. This must be an absolute path (`/opt/nomad/license.hclic`). The
-  license can also be set by setting `NOMAD_LICENSE_PATH` or by setting
-  `NOMAD_LICENSE` as the entire license value. `license_path` has the highest
-  precedence, followed by `NOMAD_LICENSE` and then `NOMAD_LICENSE_PATH`.
+- `heartbeat_grace` `(string: "10s")` - Specifies the additional time given
+  beyond the heartbeat TTL of Clients to account for network and processing
+  delays and clock skew. This is specified using a label suffix like "30s" or
+  "1h". See [below](#client-heartbeats) for details.
 
 - `min_heartbeat_ttl` `(string: "10s")` - Specifies the minimum time between
-  node heartbeats. This is used as a floor to prevent excessive updates. This is
-  specified using a label suffix like "30s" or "1h". Lowering the minimum TTL is
-  a tradeoff as it lowers failure detection time of nodes at the tradeoff of
-  false positives and increased load on the leader.
+  Client heartbeats. This is used as a floor to prevent excessive updates. This
+  is specified using a label suffix like "30s" or "1h". See
+  [below](#client-heartbeats) for details.
 
-- `failover_heartbeat_ttl` `(string: "5m")` - Specifies the TTL applied to
-	heartbeats after a new leader is elected, since we no longer know the status
-	of all the heartbeats. This is specified using a label suffix like "30s" or
-	"1h".
-
-  ~> Lowering the `failover_heartbeat_ttl` is a tradeoff as it lowers failure
-  detection time of nodes at the tradeoff of false positives. False positives
-  could cause all clients to stop their allocations if a leadership transition
-  lasts longer than `heartbeat_grace + failover_heartbeat_ttl`.
+- `failover_heartbeat_ttl` `(string: "5m")` - The time by which all Clients
+  must heartbeat after a Server leader election. This is specified using a label
+  suffix like "30s" or "1h". See [below](#client-heartbeats) for details.
```

Comment:
Including the name of the header section helps orient the reader once they land (particularly for folks with non-default text zoom settings for reading assistance):

```suggestion
  suffix like "30s" or "1h". See [Client Heartbeats](#client-heartbeats) below for details.
```

Reply:
Thanks! Fixed up all the references.